velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -452,6 +452,39 @@ VeloxToSubstraitExprConvertor::toSubstraitExpr(
   return *substraitFieldExpr;
 }
 
+const ::substrait::Expression&
+VeloxToSubstraitExprConvertor::toSubstraitSingularOrList(
+    google::protobuf::Arena& arena,
+    const std::vector<core::TypedExprPtr>& inputs,
+    const RowTypePtr& inputType,
+    ::substrait::Expression* substraitExpr) {
+  ::substrait::Expression_SingularOrList* singularOrList =
+      substraitExpr->mutable_singular_or_list();
+  VELOX_CHECK(inputs.size() == 2, "Two arguments expected for IN.");
```

Comment:
Use VELOX_CHECK_EQ for better error messages.

Reply:
Fixed.